makefile

```diff
@@ -39,12 +39,16 @@ ldFlags := "$(if $(DEBUG_ENABLED),,-w -s )-X=github.com/evergreen-ci/evergreen.B
 karmaFlags := $(if $(KARMA_REPORTER),--reporters $(KARMA_REPORTER),)
 # end evergreen specific configuration
 
+gopath := $(shell go env GOPATH)
+ifeq ($(OS),Windows_NT)
+gopath := $(shell cygpath -m $(gopath))
+endif
 
 # start linting configuration
 #   package, testing, and linter dependencies specified
 #   separately. This is a temporary solution: eventually we should
 #   vendorize all of these dependencies.
-lintDeps := github.com/alecthomas/gometalinter
+lintDeps := github.com/alecthomas/gometalinter github.com/richardsamuels/evg-lint/...
```

Comment:
I'd put this on a seperate line, though it doesn't really matter 

I don't think you need the ...

Reply:
I forked golang/lint for this, which stores the main package in golang/lint/golint. So, for my linter, it's in richardsamuels/evg-lint/evg-lint. To download/install the binary, I found the ... was necessary.

Reply:
why did you need to fork this?

Reply:
Linters are just executables that take go files/packages. There is a lot of stuff around that like importing packages, parsing flags, etc. 

For reference I wrote 138 lines of linting code, but preserved ~1100 lines of other functionality from golint around parsing.